aea/protocols/dialogue/base.py

```diff
@@ -1210,30 +1227,36 @@ def _get_collection_instance(self, col_name: str) -> Optional[SyncCollection]:
             return None
         return self._skill_component.context.storage.get_sync_collection(col_name)
 
-    def _get_collections(
-        self,
-    ) -> Tuple[Optional[SyncCollection], Optional[SyncCollection]]:
-        """Get active state dialogue collection and terminal collection."""
+    @cached_property
+    def _terminal_dialogues_collection(self) -> Optional[SyncCollection]:
         col_name = self._get_collection_name()
         if not col_name:
-            return (None, None)  # pragma: nocover
-        termina_col_name = f"{col_name}_terminal"
+            return None
+        col_name += "_terminal"
```

Comment:
define constant elsewhere and reuse here

Reply:
done